### DISCUSSION TITLE
Hyphenate or change lines

### DISCUSSION THREAD

**Discussion by weinihou** [richshaw2015/dino-rss-electron#4] [Ideas]
The words at the end of lines could be seperated to two lines, **without hyphen**. Maybe some auto-adjustment strategies should be added: hyphenate within a word or change lines out of the word.

The version I use is the MacOS 0.3.0. 
Thanks again for this program.

**Comment by richshaw2015** [comment]
Do you have any screenshots?

**Reply by weinihou** [reply]
![Screen Shot 2021-04-13 at 11 02 40 PM](https://user-images.githubusercontent.com/24472499/114575035-618ef980-9cac-11eb-8ac8-369a513f3bc5.png)

Here is the screenshot.
The words suffer both in title and body.

**Reply by richshaw2015** [reply]
Fixed at v0.3.2, you can get it from here [https://dinorss.org](https://dinorss.org/)
